hack/test-extended.sh

```diff
@@ -144,12 +147,21 @@ wait_for_url "https://${OS_MASTER_ADDR}/api/v1beta1/minions/127.0.0.1" "" 0.25 8
 # Start the Docker registry (172.30.17.101:5000)
 start_docker_registry
 
-wait_for_command '[[ "$(osc get endpoints docker-registry -t "{{ if .endpoints}}{{ len .endpoints }}{{ else }}0{{ end }}" 2>/dev/null || echo "0")" != "0" ]]' $((5*TIME_MIN))
+# TODO: simplify when #4702 is fixed upstream
+wait_for_command '[[ "$(osc get endpoints docker-registry --output-version=v1beta1 -t "{{ if .endpoints }}{{ len .endpoints }}{{ else }}0{{ end }}" || echo "0")" != "0" ]]' $((5*TIME_MIN))
```

Comment:
I would definitely agree, but not knowing about changes in v1beta3 I was not using it.
I am doing the changes now.